packages/react-instantsearch/package.json

```diff
@@ -32,14 +32,9 @@
     "algoliasearch-helper": "^2.21.0",
     "classnames": "^2.2.5",
     "lodash": "^4.17.4",
-    "prop-types": "^15.5.10"
-  },
-  "devDependencies": {
-    "enzyme": "3.3.0",
-    "enzyme-adapter-react-16": "1.1.1",
```

Comment:
If you have some dependencies that are only worth it for some packages you can also leave them in the package itself. What I am used to do is to have only the release dependencies (lerna) at the package root level while keeping every package self-contained and still be able to run npm test in every package.

wdyt?

Reply:
I saw a lots of different approach regarding that and I'm still not very convinced which one is the best fit. For us since we won't have a lots of packages (4-5 unlike Babel for example) your solution sounds legit. At the end we keep in the package dependencies all the one required by the scripts right? 

For example `babel` will be used by the `build` command so we should put it in **every** packages. On the other hand `eslint` will be run globally from the top level package so we can keep it only on the top level. Note that for the `babel` case if we want to correctly follow the self-contained rule we should have a `.babelrc` on every level also.

The other idea is to keep all the `devDependencies` on the top level package. So the packages only have their own `dependencies`. We break the self-contained rule but at the end it's a monorepo and everything is tight together.

WDYT?

Reply:
probably best to list it under devDeps in every “package”, so that you can pretend it’s its own package without the workspace too, seems the most consistent

Reply:
I'll go for the solution that your proposed, having self-contained packages since we won't have hundred of packages it makes more sense. Thanks for the feedback!